app/assets/javascripts/oxalis/model/actions/actions.js

```diff
@@ -18,7 +19,8 @@ export type ActionType =
   | TaskActionType
   | SaveActionType
   | ViewModeActionType
-  | AnnotationActionTypes;
+  | AnnotationActionTypes
+  | FlycamActionType;
```

Comment:
Good catch! It's a bummer that flow didn't error before, apparently? I guess the dispatch typing is not good enough for that?

Reply:
I think the problem is that if flow doesn't know a specific action type `"MY_ACTION"` and it sees a `case "MY_ACTION":` it stops type checking as it thinks this case is never going to be executed, right?
Would be nice if flow complained when it encounters a non-reachable case. (if I understand this correctly)